src/content/docs/en/recipes/minification.mdx

```diff
@@ -0,0 +1,61 @@
+---
+title: Minify HTML responses with middleware
+description: Use Astro middleware to minify HTML responses
+i18nReady: false
+type: recipe
+---
+
+import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
+
+Astro supports built-in [HTML minification](/en/reference/configuration-reference/#compresshtml) which works for `.astro` components, but not for [islands](/en/concepts/islands/). This recipe shows how to use Astro middleware to minify complete HTML responses.
+
```

Comment:
```suggestion

## Prerequisites

Any existing Astro project.
```
I still like having a prerequisites section, and being explicit that there are no constraints on how your project works! SSG/SSR/content collections etc. Or, if there *is* anything required, stick it there. Middleware isn't experimental anymore, so that's not a requirement.